test/units/TEST-07-PID1.startv.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: LGPL-2.1-or-later
+# shellcheck disable=SC2016
+set -eux
+set -o pipefail
+
+# shellcheck source=test/units/util.sh
+. "$(dirname "$0")"/util.sh
+
+systemd-run -v --wait echo wampfl | grep wampfl
+
+systemd-run -v -p Type=notify bash -c 'echo brumfl ; systemd-notify --ready ; echo krass' |  grep brumfl
+
+mkdir -p /etc/systemd/journald.conf.d/
+
+# Let's disable storage of debug messages, since we want to flood the journal
+# daemon with messages that it will have to process, but we do not actually
+# want to push out our own messages from storage while doing so
+cat >> /etc/systemd/journald.conf.d/50-disable-debug.conf <<EOF
```

Comment:
Please copy to /run/ and edit there, as we want to avoid touching the rootfs in tests